src/main/kotlin/no/nav/familie/ef/personhendelse/inntekt/Inntektsendringer.kt

```diff
@@ -0,0 +1,54 @@
+package no.nav.familie.ef.personhendelse.inntekt
+
+import no.nav.familie.ef.personhendelse.client.OppgaveClient
+import no.nav.familie.ef.personhendelse.inntekt.vedtak.EfVedtakRepository
+import org.slf4j.Logger
+import org.slf4j.LoggerFactory
+import org.springframework.stereotype.Component
+import java.time.YearMonth
+
+@Component
+class Inntektsendringer(
```

Comment:
`Inntektsendringer` høres ut som ett domeneobjekt :P `InntektsendringerService`? 
Er det noen spesiell grunn til å bruke Component i stedet for Service her? Har veldig lite å si egentligen, då det i prinsipp er den samme saken, men tenkte mest då vi bruker Service mye i ef-sak